components/RegisterISCN.vue

```diff
@@ -221,6 +232,25 @@ const typeOptions = [
   { label: 'CreativeWork', value: 'CreativeWork' }
 ]
 
+const licenseMap: { [key: string]: string | null } = {
+  'Copyright. All rights reserved.': 'All Rights Reserved',
+  'CC BY-NC-ND': 'https://creativecommons.org/licenses/by-nc-nd/4.0/',
+  'CC BY-NC-SA': 'https://creativecommons.org/licenses/by-nc-sa/4.0/',
+  'CC BY-NC': 'https://creativecommons.org/licenses/by-nc/4.0/',
+  'CC BY-ND': 'https://creativecommons.org/licenses/by-nd/4.0/',
+  'CC BY-SA': 'https://creativecommons.org/licenses/by-sa/4.0/',
+  'CC BY': 'https://creativecommons.org/licenses/by/4.0/',
+  'CC0 (Public Domain)': 'https://creativecommons.org/publicdomain/zero/1.0/',
+  Other: null
+}
+
+const licenseOptions = Object.entries(licenseMap).map(([label, value]) => ({
+  label,
+  value: value || label
+}))
```

Comment:
Just set `Other` as value so that we can have a consistent type of Map here? We will overide with `customLicense` value anyway

Reply:
[💄 Enhance license selection](https://github.com/likecoin/nft-book-press/pull/292/commits/05bd4e9baa192a478e8a81cee8028a2b58d6f05c)